spark_auto_mapper_fhir/backbone_elements/immunization_performer_backbone_element.py

```diff
@@ -0,0 +1,37 @@
+from typing import Optional, Union
+
+from spark_auto_mapper_fhir.complex_types.reference import Reference
+from spark_auto_mapper_fhir.fhir_types.id import FhirId
+from spark_auto_mapper_fhir.fhir_types.list import FhirList
+from spark_auto_mapper_fhir.backbone_elements.fhir_backbone_element_base import (
+    FhirBackboneElementBase,
+)
+from spark_auto_mapper_fhir.resources.organization import Organization
+from spark_auto_mapper_fhir.resources.practitioner import Practitioner
+from spark_auto_mapper_fhir.resources.practitioner_role import PractitionerRole
+from spark_auto_mapper_fhir.valuesets.immunization_function import (
+    ImmunizationFunctionCode,
+)
+
+
+class ImmunizationPerformerBackboneElement(FhirBackboneElementBase):
+    def __init__(
+        self,
+        *,
+        id_: Optional[FhirId] = None,
+        function: Optional[FhirList[ImmunizationFunctionCode]] = None,
```

Comment:
missing CodeableConcept between FhirList and type code

Reply:
Corrected